src/main/resources/static/js/MainWindow.js

```diff
@@ -1,769 +1,743 @@
+              }
+            );
+            stop = true;
+            return false;
+          }
+
+          parsed = parseInt(currentInput, 10);
+
+          if (!(currentInput == parsed) || parsed < 0) {
+            $.notify(
+              {
+                message:
+                  "Please specify the column where the observations of each datastream can be found (must be a non-negative number) before saving the configuration."
+              },
+              {
+                allow_dismiss: true,
+                type: "info",
+                placement: { from: "top", align: "left" },
+                animate: {
+                  enter: "animated fadeInDown",
+                  exit: "animated fadeOutUp"
+                }
+              }
+            );
+            stop = true;
+            return false;
+          }
+          obs.push(parsed);
+        });
+      if (stop == true) {
+        return false;
+      }
+
+      obj["observations"] = obs;
+      obj["multiStream"] = obs.length > 1;
+      streams.push(obj);
+    });
+
+  if (stop == true) {
+    return false;
+  }
+
+  var filetype = null;
+  if ($("input[name=source]:eq(0)").is(":checked")) {
+    filetype = $("#file").val();
+  } else if ($("input[name=source]:eq(1)").is(":checked")) {
+    filetype = $("#sourceinput").val();
+  }
+  if (!filetype || filetype == "") {
+    addToLog("No File specified.");
+  } else {
+    filetype = filetype
+      .split(".")
+      .pop()
+      .toUpperCase();
+    if (filetype == "XLSX" || filetype == "XLS") {
+      filetype = "EXCEL";
+    } else if (filetype !== "CSV") {
+      addToLog("Unknown file type.");
+      return;
+    }
+  }
+
+  if ($("#selecttime option:selected").attr("data-value") == null) {
+    $.notify(
+      {
+        message: "Please choose a time zone before saving the configuration."
+      },
+      {
+        allow_dismiss: true,
+        type: "info",
+        placement: { from: "top", align: "left" },
+        animate: { enter: "animated fadeInDown", exit: "animated fadeOutUp" }
+      }
+    );
+    return false;
+  } else if (date.length == 0) {
+    $.notify(
+      {
+        message:
+          "Please specify where to find the date in your file before saving the configuration."
+      },
+      {
+        allow_dismiss: true,
+        type: "info",
+        placement: { from: "top", align: "left" },
+        animate: { enter: "animated fadeInDown", exit: "animated fadeOutUp" }
+      }
+    );
+    return false;
+  } else if (streams.length == 0) {
+    $.notify(
+      { message: "Please add a datastream before saving the configuration." },
+      {
+        allow_dismiss: true,
+        type: "info",
+        placement: { from: "top", align: "left" },
+        animate: { enter: "animated fadeInDown", exit: "animated fadeOutUp" }
+      }
+    );
+    return false;
+  }
+
+  var map = mappingData;
+
+  var cfgName = prompt("Name of Configuration:");
+  if (cfgName == null) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4898788)